lib/utils/date_time_util.dart

```diff
@@ -41,10 +42,12 @@ Map<int, String> _days = {
 };
 
 String getMonthAndYear(DateTime dateTime) {
+  //Jun 2022
```

Comment:
Nice! Can you add these these above the method so it can act as quick documentation for the method? The IDE will suggest/hint the return format/